api/wasm/cpp/proxy_wasm_api.h

```diff
@@ -355,9 +355,7 @@ class RootContext : public ContextBase {
   const std::string root_id_;
   std::unordered_map<uint32_t, HttpCallCallback> http_calls_;
   std::unordered_map<uint32_t, GrpcSimpleCallCallback> simple_grpc_calls_;
-  std::unique_ptr<GrpcCallHandlerBase> cur_grpc_call_;
   std::unordered_map<uint32_t, std::unique_ptr<GrpcCallHandlerBase>> grpc_calls_;
-  std::unique_ptr<GrpcStreamHandlerBase> cur_grpc_stream_;
```

Comment:
You shouldn't be reverting those changes, see: https://github.com/istio/envoy/pull/175#issuecomment-594182783

Reply:
But we didn't ship that in 1.4.6.  I think instead we should get release-1.4.6 into a state that looks like the release we just shipped, tag it, then reapply any fixes which can go out in 1.4.7

@howardjohn CC

Reply:
That's a valid point. OK, fair enough, no one should be using Wasm in 1.4 anyway...

cc @mandarjog @kyessenov @bianpengyuan for visibility.

Reply:
I disagree that we need to get release-1.4 branch into a state that looks like 1.4.6. We did not ship from 1.4 branch, as in reality we created a private 1.4.6 branch that was developed independently from 1.4 branch. Now that 1.4.6 branch is done, we should simply merge 1.4.6 branch into 1.4 branch (and that was exceptional due to secrecy around the patches).